### PR TITLE
Dry run support (with json output)

### PR DESCRIPTION
**Pull request by 1is10**
https://github.com/krzysztofzablocki/Sourcery/issues/664

### CODE REVIEW

File: SourceryTests/Models/TypedSpec.generated.swift [hunk 1]
```diff
@@ -315,4 +315,4 @@ class TypedSpec: QuickSpec {
             }
         }
     }
-}
+}
```
Comment (krzysztofzablocki) [inline SourceryTests/Models/TypedSpec.generated.swift:318]: where is the lack of empty EOF line coming from? do you have some post-processing that happens outside of Sourcery?

**Review by krzysztofzablocki — COMMENTED** [review]
looking good

File: SourceryTests/Models/TypedSpec.generated.swift (hunk 1 above)
Comment (1is10) [reply SourceryTests/Models/TypedSpec.generated.swift]: Sorry, my bad, i will be more careful in future. Cause of this, is that i run temporary version of sourcery by mistake inside repo. 

That was early version that trims end lines, that bug don't exist in current version.

i rechecked code, removed changes to package.resolved & regenerated SourceryRuntime code with sourcery binary.

**Comment by krzysztofzablocki** [comment]
thanks for contributing 🙇

**Comment by 1is10** [comment]
Thank you for sourcery 🙌

**Pull request merged**